src/api/models.py

```diff
@@ -242,534 +238,1344 @@ class ShotModel(SQLModel, table=True):
 
-    cpf_p05009: Optional[float] = Field(nullable=True)
+    cpf_p05009: Optional[float] = Field(alias="mcs_select_mfps")
 
-    cpf_p05010: Optional[float] = Field(nullable=True)
+    cpf_p05010: Optional[float] = Field(alias="mcs_select_efps")
 
-    cpf_p05011: Optional[float] = Field(nullable=True)
+    cpf_p05011: Optional[float] = Field(alias="mcs_select_sfps")
 
-    cpf_p05015: Optional[float] = Field(nullable=True)
+    cpf_p05015: Optional[float] = Field(alias="mcs_select_scs4")
 
-    cpf_p05016: Optional[float] = Field(nullable=True)
+    cpf_p05016: Optional[float] = Field(alias="mcs_select_fa3")
 
-    cpf_p05017: Optional[float] = Field(nullable=True)
+    cpf_p05017: Optional[float] = Field(alias="mcs_select_fa4")
 
-    cpf_p05025: Optional[float] = Field(nullable=True)
+    cpf_p05025: Optional[float] = Field(alias="mcs_select_p1ps")
 
-    cpf_p05027: Optional[float] = Field(nullable=True)
+    cpf_p05027: Optional[float] = Field(
+        alias="mcs_select_cp3c",
+        description="(MCS Setup) select P3 counterpulse bank for MAST only",
+    )
 
-    cpf_p05028: Optional[float] = Field(nullable=True)
+    cpf_p05028: Optional[float] = Field(
+        alias="mcs_select_cp4",
+        description="(MCS Setup) select P4 start bank for MAST only",
+    )
 
-    cpf_p05029: Optional[float] = Field(nullable=True)
+    cpf_p05029: Optional[float] = Field(
+        alias="mcs_select_cp5",
+        description="(MCS Setup) select P3 start bank for MAST only",
+    )
 
-    cpf_p05030: Optional[float] = Field(nullable=True)
+    cpf_p05030: Optional[float] = Field(
+        alias="mcs_select_p3p_s",
+        description="(MCS Setup) select Siemens & GEC switches for P3 plasma Initiation for MAST only",
+    )
 
-    cpf_p05032: Optional[float] = Field(nullable=True)
+    cpf_p05032: Optional[float] = Field(
+        alias="mcs_select_p4_thy",
+        description="(MCS Setup) select P4 Ignitron and Thyristor circuit for MAST only",
+    )
 
-    cpf_p05033: Optional[float] = Field(nullable=True)
+    cpf_p05033: Optional[float] = Field(
+        alias="mcs_select_p5_thy",
+        description="(MCS Setup) select P5 Ignitron and Thyristor circuit for MAST only",
+    )
 
-    cpf_p05153: Optional[float] = Field(nullable=True)
+    cpf_p05153: Optional[float] = Field(alias="mcs_select_ecrh")
 
-    cpf_p06000: Optional[float] = Field(nullable=True)
+    cpf_p06000: Optional[float] = Field(alias="mcs_select_xmm_os9")
 
-    cpf_p06001: Optional[float] = Field(nullable=True)
+    cpf_p06001: Optional[float] = Field(alias="mcs_select_xpc_os9")
 
-    cpf_p06002: Optional[float] = Field(nullable=True)
+    cpf_p06002: Optional[float] = Field(alias="mcs_select_xcm_os9")
 
-    cpf_p06003: Optional[float] = Field(nullable=True)
+    cpf_p06003: Optional[float] = Field(alias="mcs_select_xmw_os9")
 
-    cpf_p06004: Optional[float] = Field(nullable=True)
+    cpf_p06004: Optional[float] = Field(alias="mcs_select_xma_os9")
```

Comment:
Do these have any description in the CPF website? If not we should chase Jimmy  Measures for the description.

Reply:
Well the description on the web is just the same as the name given. So I decided it was not really necessary. But I have added it with the addition of "for MAST only"